app/forms/state_file/esign_declaration_form.rb

```diff
@@ -3,15 +3,26 @@ class EsignDeclarationForm < QuestionsForm
     set_attributes_for :state_file_efile_device_info, :device_id
     set_attributes_for :intake,
                        :primary_esigned,
-                       :spouse_esigned
+                       :spouse_esigned,
+                       :primary_signature_pin,
+                       :spouse_signature_pin
 
     validates :primary_esigned, acceptance: { accept: 'yes', message: ->(_object, _data) { I18n.t("views.ctc.questions.confirm_legal.error") }}
     validates :spouse_esigned, acceptance: { accept: 'yes', message: ->(_object, _data) { I18n.t("views.ctc.questions.confirm_legal.error") }}, if: -> { @intake.ask_spouse_esign? }
```

Comment:
These copy keys are from CTC and are in danger of being deleted later. It would be better to move them somewhere generic or copy them over to an appropriate state file key.